tests/walkthroughs/sdk-script-manual-tests/README.md

```diff
@@ -1,5 +1,5 @@
 ### NOTE
-Before running tests, `.global.json` files should be renamed to `global.json`, this is needed to not confuse UseDotNet task of Azure Pipelies when installing SDK.
+Before running tests, `.global.json` files should be renamed to `global.json`, this is needed to not confuse UseDotNet task of Azure Pipelines when installing SDK.
```

Comment:
I like pipelies actually :D

Reply:
<details><summary>I think I like that better than Azure Pipelines 😉</summary>

<img width="378" alt="image" src="https://github.com/dotnet/fsharp/assets/2119212/d581ccf0-1ee4-4ec0-a8dc-28eeb082fe03">

![image](https://github.com/dotnet/fsharp/assets/2119212/3e30fe56-87ca-4597-8a91-8404b19ca474)

</details>

Anyway, I have one commit to fix some formatting that I'll push in ~10-15 minutes, I'm hoping that'll be the last fix for CI.